gossip/src/cluster_info.rs

```diff
@@ -3920,7 +3923,7 @@ mod tests {
                 timestamp(),
                 None,
                 &HashMap::new(),
-                MAX_BLOOM_SIZE,
+                992, // max_bloom_filter_bytes
```

Comment:
where did you get 992 from? from: `PACKET_DATA_SIZE - 4 - serialized_size(LegacyContactInfo)`?

Reply:
This is a test so it is arbitrary and it does not matter, but I took 992 from the last case of `test_get_max_bloom_filter_bytes` test.
Old code was using `MAX_BLOOM_SIZE == 928`.